### PR TITLE
add a default duration for when end not specified

### PR DESCRIPTION
**Pull request by enderv**
Addresses https://github.com/code4recovery/tsml-ui/issues/245

### CODE REVIEW

**Comment by joshreisner** [comment]
thanks! i will check this out with sites currently using TSML UI who are not supplying end times, to see if they are comfortable with this update

changes look great overall, some small bits of feedback:

* it would be great if you could format with Prettier, looks like at least one of the files would be affected
* i don't think we need the google sheet change - the purpose of that function is to prepare the `end_time` for processing in the main load-meeting-data function, so in the `HH:MM` text format basically
* in format-ics i think we can just remove the end_time logic now, since meetings with a start will also now have an end

**Comment by enderv** [comment]
can got prettier installed and formatted, undid the google change but wasn't sure what to do on "in format-ics i think we can just remove the end_time logic now"

Also thoughts on an issue to add prettier as dev dependency and npm command to run it?

File: src/helpers/format/format-ics.ts [hunk 1]
```diff
@@ -9,9 +10,9 @@ export function formatIcs(meeting: Meeting) {
 
   if (!meeting.start) return;
 
-  //need an end time. guess one hour if none specified
+  //need an end time. use default if none specified
   if (!meeting.end) {
-    meeting.end = meeting.start.plus({ hour: 1 });
+    meeting.end = meeting.start.plus({ minutes: settings.defaults.duration });
   }
```
Comment (joshreisner) [inline src/helpers/format/format-ics.ts:16]: this is what i was referencing. now that we are supplying `end` times, i think this set of lines can now just be 

```
if (!meeting.start || !meeting.end) return;
```

**Comment by joshreisner** [comment]
left a comment on `format-ics`

sounds great on creating an issue to add prettier as dev dependency and npm command to run it -- i wasn't aware that was possible

**Comment by enderv** [comment]
Oops sorry been swamped last two days got the issue in though. Any other fixes needed after your commit?

**Comment by joshreisner** [comment]
Hey, no, should be all good now. I just have to give AA SF / Marin a few days to implement end times if they want to. Looking to merge on Sunday / Monday

**Pull request merged**